spring-cloud-netflix-eureka-client/src/main/java/org/springframework/cloud/netflix/eureka/EurekaClientAutoConfiguration.java

```diff
@@ -120,6 +123,7 @@ public EurekaInstanceConfigBean eurekaInstanceConfigBean(InetUtils inetUtils) {
 		EurekaInstanceConfigBean instance = new EurekaInstanceConfigBean(inetUtils);
 		instance.setNonSecurePort(this.nonSecurePort);
 		instance.setInstanceId(getDefaultInstanceId(this.env));
+		instance.setPreferIpAddress(preferIpAddress);
```

Comment:
This should get automatically set since it is simply a `@ConfigurationProperties` bean. The other items set here have business logic performed.